ui/analyse/src/study/multiBoard.ts

```diff
@@ -29,7 +29,7 @@ export class MultiBoardCtrl {
     readonly redraw: () => void,
   ) {
     this.playing = toggle(false, this.redraw);
-    this.showResults = toggle(true, this.redraw);
+    this.showResults = storedBooleanProp("study.showResults", true);
```

Comment:
`this.redraw` has been dropped? it wasn't necessary?

Reply:
It seems to be working fine without it, I will test it more to make sure it's not needed.